aicsimageio/readers/arraylike_reader.py

```diff
@@ -52,20 +52,6 @@ def _read_immediate(self):
     def dims(self) -> str:
         return self._dims
 
-    @dims.setter
```

Comment:
Does this mean telling the reader the dims is no longer an option or has that been shifted into another function?

Reply:
> removed all custom dimension dims setting from base readers

Yes. Custom dims setting has been removed. I tried to implement that for all readers but it ended up being more complicated.

From resolved conversation:
> After discussion with @toloudis, I have actually decided to remove dimension setting entirely from the base readers. You can still inform the AICSImage object of custom dimensions with AICSImage(..., known_dims="BLAH") but the reader objects are to some level the raw data (even with our best guessing on certain readers)